zulipterminal/ui_tools/buttons.py

```diff
@@ -19,10 +19,12 @@ def __init__(self, controller: Any, caption: str,
                  show_function: Callable[..., Any], width: int,
                  prefix_character: Union[str, Tuple[Any, str]]='\N{BULLET}',
                  text_color: Optional[str]=None,
-                 count: int=0) -> None:
+                 count: int=0, stream: str='', topic: str='') -> None:
         self.caption = caption
         self.prefix_character = prefix_character
         self.count = count
+        self.title = topic
+        self.stream = stream
```

Comment:
It seems like these two could just go into the `TopicButton` class?

Reply:
Ah, yes. I forgot to update this. Have changed it in the new version.